### PR TITLE
Bootstrap and dependencies are updated

### PR DESCRIPTION
**Pull request by solomax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ How to get help and news
 
 * Keep up to date on announcements and more by following me [@l0rdn1kk0n](http://twitter.com/l0rdn1kk0n) on Twitter or use the [#WicketBootstrap](https://twitter.com/search?q=%23WicketBootstrap&src=typd) hashtag
 * use the [google group](https://groups.google.com/d/forum/wicket-bootstrap) (wicket-bootstrap (at) googlegroups.com)
-* you can see wicket-bootstrap in action: https://wb8.teliclab.info
+* you can see wicket-bootstrap in action: http://wb.agilecoders.de/
```
Comment (martin-g) [inline README.md:65]: Let's just remove this.
http://wb.agilecoders.de/ now forwards to https://doridina.de/

File: bootstrap-extensions/pom.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
     <properties>
         <inputmask.version>4.0.0</inputmask.version>
-        <jqueryui.version>1.12.1</jqueryui.version>
+        <jqueryui.version>1.13.0</jqueryui.version>
```
Comment (martin-g) [inline bootstrap-extensions/pom.xml:18]: JQueryUI is still alive ! Interesting!

**Review by martin-g — APPROVED** [review]

**Comment by martin-g** [comment]
Thank you, @solomax !

File: README.md (hunk 1 above)
Comment (solomax) [reply README.md:65]: @martin-g you too fast :)
Shall this be removed in separate PR? :)

File: bootstrap-extensions/pom.xml (hunk 2 above)
Comment (solomax) [reply bootstrap-extensions/pom.xml:18]: I was surprised :)))

File: README.md (hunk 1 above)
Comment (martin-g) [reply README.md:65]: I decided to do it myself and not bother you :-)

**Pull request merged**